pandas/core/frame.py

```diff
@@ -3402,11 +3402,15 @@ def stack(self, level=-1, dropna=True):
 
         Parameters
         ----------
-        level : int, string, or list of these, default last level
-            Level(s) to stack, can pass level name
+        level : int, string, list of these, or None; default -1 (last level)
+            Level(s) to stack, can pass level name(s).
+            None specifies all column levels, i.e. list(range(columns.nlevels)).
         dropna : boolean, default True
             Whether to drop rows in the resulting Frame/Series with no valid
             values
+        sequentially : boolean, default True
+            When level is a list (or None), whether the multiple column levels
+            should be stacked sequentially (if True) or simultaneously (if False).
 
         Examples
         ----------
```

Comment:
maybe add to the examples here for use of ``level=None`` and ``sequentially=False`` (might be too big for here though....), will need to update docs as well for these features (e.g. you can also have a pointer to the docs)